rai/node/openclwork.cpp

```diff
@@ -383,7 +383,7 @@ __kernel void raiblocks_work (__global ulong * attempt, __global ulong * result_
 void printstate (blake2b_state * S)
 {
 	printf ("%lu %lu %lu %lu %lu %lu %lu %lu %lu %lu %lu %lu\n", S->h[0], S->h[1], S->h[2], S->h[3], S->h[4], S->h[5], S->h[6], S->h[7], S->t[0], S->t[1], S->f[0], S->f[1]);
```

Comment:
I know this is just a debug dump, but... `%lu` isn't exactly portable, maybe use `PRIu64` or replace the whole thing with `std::cout` ?